### PR TITLE
:memo: Include `Mac Address` documentation

### PR DESCRIPTION
**Pull request by yezz123**
## Change Summary

I just merged this pull request regarding Mac Address new type, and we need to include documentation https://github.com/pydantic/pydantic-extra-types/pull/71

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @hramezani

### CODE REVIEW

**Comment by yezz123** [comment]
please review

**Comment by adriangb** [comment]
cc @tpdorsey

File: docs/usage/types/extra_types/extra_types.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ The following types are supported by [Pydantic Extra Types]:
 * [Payment Card Numbers](payment_cards.md) &mdash; a type that allows you to store payment card numbers in your model.
 * [Phone Numbers](phone_numbers.md) &mdash; a type that allows you to store phone numbers in your model.
 * [Routing Numbers](routing_numbers.md) &mdash; a type that allows you to store ABA transit routing numbers in your model.
+* [Mac Address](mac_address.md) &mdash; a type that allows you to store Mac Address in your model.
```
Comment (tpdorsey) [inline docs/usage/types/extra_types/extra_types.md:11]:
```suggestion
* [MAC Address](mac_address.md) &mdash; a type that allows you to store MAC addresses in your model.
```

Comment (tpdorsey) [reply docs/usage/types/extra_types/extra_types.md:11]: You may want to update the docs for your extra types PR with this usage. Feel free to tag me for review.

File: mkdocs.yml [hunk 2]
```diff
@@ -83,6 +83,7 @@ nav:
       - Payment Card Numbers: usage/types/extra_types/payment_cards.md
       - Phone Numbers: usage/types/extra_types/phone_numbers.md
       - Routing Numbers: usage/types/extra_types/routing_numbers.md
+      - Mac Address: usage/types/extra_types/mac_address.md
```
Comment (tpdorsey) [inline mkdocs.yml:86]:
```suggestion
      - MAC Address: usage/types/extra_types/mac_address.md
```

**Review by tpdorsey — APPROVED** [review]

**Comment by tpdorsey** [comment]
@yezz123 thanks for the contribution! See notes for some minor edits. You may want to apply these edits to your docs as well for correct usage of "MAC address". Cheers

**Comment by yezz123** [comment]
Hello @tpdorsey I opened a PR https://github.com/pydantic/pydantic-extra-types/pull/72 and add you as reviewer when its ready we can merge it both

Thank you 🙏🏻

**Review by hramezani — APPROVED** [review]

**Comment by hramezani** [comment]
Thanks @yezz123

**Pull request merged**